lib/diagnostic.rb

```diff
@@ -22,7 +22,7 @@
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = 'star wars episode viii.'
 
```

Comment:
remember we use snake case when naming a variable in Ruby.
we could define it as `star_wars_episode_viii`